src/pages/pokedex-site.tsx

```diff
@@ -71,8 +71,10 @@ export default function PokedexSite() {
 
     return (
         <Layout>
-            <div className="grid grid-cols-3 gap-2 gap-x-12">
-                <div className="flex flex-col gap-2">
+            {/* main content wrapper  */}
+            <div className="grid desktop:grid-cols-12 grid-cols-4 gap-4 col-span-full content-start gap-x-12">
```

Comment:
If you actually want a different gap on both axises, you should explicitely use gap-x and gap-y.